Assets/PatchKit Patcher/Scripts/AppData/Local/Pack1Unarchiver.cs

```diff
@@ -274,42 +274,45 @@ private Stream CreateGzipDecompressor(Stream source)
         {
             using (var cryptoStream = new CryptoStream(sourceStream, decryptor, CryptoStreamMode.Read))
             {
-                using (Stream decompressionStream = createDecompressor(cryptoStream))
+                using (var wrapperStream = new GZipReadWrapperStream(sourceStream))
```

Comment:
It would be better to move it to `CreateGzipDecompressor`

Reply:
The problem with that is that it we wouldn't be able to properly dispose of the wrapper stream.
That wouldn't technically be a problem, since the wrapper stream doesn't need to be disposed.
But it would be a pretty severe code inconsistency.